contracts/Tokens/Prime/PrimeLiquidityProvider.sol

```diff
@@ -0,0 +1,340 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { SafeERC20Upgradeable, IERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+
+contract PrimeLiquidityProvider is AccessControlledV8 {
+    using SafeERC20Upgradeable for IERC20Upgradeable;
+
+    /// @notice The max token distribution speed
+    uint224 public constant MAX_DISTRIBUTION_SPEED = 1e18;
```

Comment:
We are doing to use a full word anyway, right? Why did you use uint224 instead uint256, out of curiosity?

Reply:
used uint256, 0ce69629170233d16184a62eaec618ea8ef3c1a5